postgresのバージョン15によるUTで一部のテストコードが失敗する。

# issueタイトル
postgresのバージョン15によるUTで一部のテストコードが失敗する。

# 環境（UT実行バージョン）
- postgres: 15.5
```
$ psql --version
psql (PostgreSQL) 15.5
```
- node: v18.16.0
```
$ fnm use 18
Using Node v18.16.0
```

# 利用バージョン
- ブランチ：develop
- コミットID：6eb14a1abc2f69175e162caa5f22ae734acdd0dd
２
# 現象
postgresのバージョン15によるUTで一部のテストコードが失敗する。

## UT実行結果
以下の2件のテストケースが失敗する。
```
Summary of all failing tests
FAIL src/tests/09-01.GetJoin.spec.ts (157.701 s)
  ● CatalogUpdate API › Region参加申請取得 › 異常：Reion-Root（template.regionが空）

    expect(received).toBe(expected) // Object.is equality

    Expected: 200
    Received: 503

      2719 |
      2720 |             // レスポンスチェック
    > 2721 |             expect(response.status).toBe(200);
           |                                     ^
      2722 |             expect(response.body).toStrictEqual([]);
      2723 |         });
      2724 |         test('異常：Reion-Root（template.regionが無い）', async () => {

      at src/tests/09-01.GetJoin.spec.ts:2721:37
      at fulfilled (src/tests/09-01.GetJoin.spec.ts:5:58)

  ● CatalogUpdate API › Region参加申請取得 › 異常：Reion-Root（template.regionが無い）

    expect(received).toBe(expected) // Object.is equality

    Expected: 200
    Received: 503

      2731 |
      2732 |             // レスポンスチェック
    > 2733 |             expect(response.status).toBe(200);
           |                                     ^
      2734 |             expect(response.body).toStrictEqual([]);
      2735 |         });
      2736 |         test('異常：カタログの取得に失敗', async () => {

      at src/tests/09-01.GetJoin.spec.ts:2733:37
      at fulfilled (src/tests/09-01.GetJoin.spec.ts:5:58)

Test Suites: 1 failed, 46 passed, 47 total
Tests:       2 failed, 639 passed, 641 total
Snapshots:   0 total
Time:        490.382 s
Ran all test suites.
```

## ログ
- logs/application.log上に出力されているエラーログ
```
[2024-03-27 20:41:40.588] [ERROR] [494a44bb97aa0ef964f6a666b9019b2d20bf05aa811919833f3e0c0ae2b09b38] [03217d75-7bda-4658-b0cd-a1858dd77f17] QueryFailedError: trailing junk after numeric literal at or near "0A"
    at PostgresQueryRunner.query (C:\Users\x-xxxxxxxxxx\Desktop\pxr-oss\pxr-catalog-update-service\src\driver\postgres\PostgresQueryRunner.ts:299:19)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at SelectQueryBuilder.loadRawResults (C:\Users\x-xxxxxxxxxx\Desktop\pxr-oss\pxr-catalog-update-service\src\query-builder\SelectQueryBuilder.ts:3789:25)
    at SelectQueryBuilder.executeEntitiesAndRawResults (C:\Users\x-xxxxxxxxxx\Desktop\pxr-oss\pxr-catalog-update-service\src\query-builder\SelectQueryBuilder.ts:3541:26)
    at SelectQueryBuilder.getRawAndEntities (C:\Users\x-xxxxxxxxxx\Desktop\pxr-oss\pxr-catalog-update-service\src\query-builder\SelectQueryBuilder.ts:1670:29)
    at SelectQueryBuilder.getMany (C:\Users\x-xxxxxxxxxx\Desktop\pxr-oss\pxr-catalog-update-service\src\query-builder\SelectQueryBuilder.ts:1760:25) {
  query: 'SELECT "joinManage"."id" AS "joinManage_id", "joinManage"."join_actor_code" AS "joinManage_join_actor_code", "joinManage"."join_actor_version" AS "joinManage_join_actor_version", "joinManage"."join_region_code" AS "joinManage_join_region_code", "joinManage"."join_region_version" AS "joinManage_join_region_version", "joinManage"."applicant_actor_code" AS "joinManage_applicant_actor_code", "joinManage"."applicant_actor_version" AS "joinManage_applicant_actor_version", "joinManage"."approval_expire_at" AS "joinManage_approval_expire_at", "joinManage"."type" AS "joinManage_type", "joinManage"."applicant_date" AS "joinManage_applicant_date", "joinManage"."is_draft" AS "joinManage_is_draft", "joinManage"."is_disabled" AS "joinManage_is_disabled", "joinManage"."created_by" AS "joinManage_created_by", "joinManage"."created_at" AS "joinManage_created_at", "joinManage"."updated_by" AS "joinManage_updated_by", "joinManage"."updated_at" AS "joinManage_updated_at", "joinServiceManage"."id" AS "joinServiceManage_id", "joinServiceManage"."join_manage_id" AS "joinServiceManage_join_manage_id", "joinServiceManage"."type" AS "joinServiceManage_type", "joinServiceManage"."service_code" AS "joinServiceManage_service_code", "joinServiceManage"."service_version" AS "joinServiceManage_service_version", "joinServiceManage"."is_disabled" AS "joinServiceManage_is_disabled", "joinServiceManage"."created_by" AS "joinServiceManage_created_by", "joinServiceManage"."created_at" AS "joinServiceManage_created_at", "joinServiceManage"."updated_by" AS "joinServiceManage_updated_by", "joinServiceManage"."updated_at" AS "joinServiceManage_updated_at", "joinApprovalManage"."id" AS "joinApprovalManage_id", "joinApprovalManage"."join_manage_id" AS "joinApprovalManage_join_manage_id", "joinApprovalManage"."auth_code" AS "joinApprovalManage_auth_code", "joinApprovalManage"."status" AS "joinApprovalManage_status", "joinApprovalManage"."comment" AS "joinApprovalManage_comment", "joinApprovalManage"."approval_actor_code" AS "joinApprovalManage_approval_actor_code", "joinApprovalManage"."approval_actor_version" AS "joinApprovalManage_approval_actor_version", "joinApprovalManage"."approver" AS "joinApprovalManage_approver", "joinApprovalManage"."approval_at" AS "joinApprovalManage_approval_at", "joinApprovalManage"."is_disabled" AS "joinApprovalManage_is_disabled", "joinApprovalManage"."created_by" AS "joinApprovalManage_created_by", "joinApprovalManage"."created_at" AS "joinApprovalManage_created_at", "joinApprovalManage"."updated_by" AS "joinApprovalManage_updated_by", "joinApprovalManage"."updated_at" AS "joinApprovalManage_updated_at" FROM "pxr_catalog_update"."join_manage" "joinManage" LEFT JOIN "pxr_catalog_update"."join_service_manage" "joinServiceManage" ON "joinServiceManage"."join_manage_id"="joinManage"."id" AND ("joinServiceManage"."is_disabled" = false)  LEFT JOIN "pxr_catalog_update"."join_approval_manage" "joinApprovalManage" ON "joinApprovalManage"."join_manage_id"="joinManage"."id" AND ("joinApprovalManage"."is_disabled" = false) WHERE "joinManage"."is_disabled" = false AND "joinManage"."applicant_actor_code" != 1000005 AND "joinManage"."join_region_code" = 0AND ("joinApprovalManage"."status" = 0 OR "joinApprovalManage"."status" IS NULL) ORDER BY "joinManage"."id" ASC',
  parameters: [],
  driverError: error: trailing junk after numeric literal at or near "0A"
      at C:\Users\x-xxxxxxxxxx\Desktop\pxr-oss\pxr-catalog-update-service\node_modules\pg\lib\client.js:526:17
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at PostgresQueryRunner.query (C:\Users\x-xxxxxxxxxx\Desktop\pxr-oss\pxr-catalog-update-service\src\driver\postgres\PostgresQueryRunner.ts:249:25)
      at SelectQueryBuilder.loadRawResults (C:\Users\x-xxxxxxxxxx\Desktop\pxr-oss\pxr-catalog-update-service\src\query-builder\SelectQueryBuilder.ts:3789:25)
      at SelectQueryBuilder.executeEntitiesAndRawResults (C:\Users\x-xxxxxxxxxx\Desktop\pxr-oss\pxr-catalog-update-service\src\query-builder\SelectQueryBuilder.ts:3541:26)
      at SelectQueryBuilder.getRawAndEntities (C:\Users\x-xxxxxxxxxx\Desktop\pxr-oss\pxr-catalog-update-service\src\query-builder\SelectQueryBuilder.ts:1670:29)
      at SelectQueryBuilder.getMany (C:\Users\x-xxxxxxxxxx\Desktop\pxr-oss\pxr-catalog-update-service\src\query-builder\SelectQueryBuilder.ts:1760:25) {
    length: 116,
    severity: 'ERROR',
    code: '42601',
    detail: undefined,
    hint: undefined,
    position: '3187',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'scan.l',
    line: '1192',
    routine: 'scanner_yyerror'
  },
  length: 116,
  severity: 'ERROR',
  code: '42601',
  detail: undefined,
  hint: undefined,
  position: '3187',
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'scan.l',
  line: '1192',
  routine: 'scanner_yyerror'
}
```
# 原因
- SQL文内に必要な半角スペースがなく、構文エラーになっている。
- 0ANDとなっているが、0 ANDと半角スペースが入るのが正しい。
```
join_region_code" = 0AND ("joinApprovalManage"."status" = 0 OR
```

--- a/src/repositories/EntityOperation.ts
+++ b/src/repositories/EntityOperation.ts
@@ -181,7 +181,7 @@ export default class EntityOperation {
                     }
                     prefix += ')';
                 } else {
-                    prefix += 'AND joinManage.joinRegionCode = 0';
+                    prefix += 'AND joinManage.joinRegionCode = 0 ';
                 }
             } else {
                 prefix += `AND joinManage.joinActorCode = ${actorCode} `;
@@ -199,7 +199,7 @@ export default class EntityOperation {
                     }
                     prefix += ')';
                 } else {
-                    prefix += 'AND joinManage.joinRegionCode = 0';
+                    prefix += 'AND joinManage.joinRegionCode = 0 ';
                 }
             } else {
                 prefix += `AND joinManage.joinActorCode = ${actorCode} `;

